idaplugin/rematch/collectors/assembly_annotation.py

```diff
@@ -0,0 +1,25 @@
+import idaapi
+import idautils
+import ida_lines
```

Comment:
ida_lines is only 6.95, can you change that to idaapi for lower version or add it to idasix?